packages/vuetify/src/directives/ripple.ts

```diff
@@ -42,6 +42,8 @@ const calculate = (e: MouseEvent, el: HTMLElement, value: RippleOptions = {}) =>
 const ripple = {
   /* eslint-disable max-statements */
   show (e: MouseEvent, el: HTMLElement, value: RippleOptions = {}) {
+    if (e.target !== e.currentTarget) return
```

Comment:
if you click on an icon inside a button, `target` is the icon element and `currentTarget` the button. Shouldn't this ripple?